golangci.yml

```diff
@@ -73,6 +73,7 @@ linters:
     - musttag
     # Deprecated ones:
     - gomnd
+    - execinquery
     # Weird/bad ones:
     - wsl
     - nlreturn
```

Comment:
Commenting random line

This one is also interesting


```yaml
linters-settings:
  usestdlibvars:
    # Suggest the use of time.Layout.
    # Default: false
    time-layout: true
```

Reply:
I don't know what that is / does :)

Reply:
let's add it in #56 which will require getting it to pass on fortio/fortio (and deps) first

I don't think I do time formatting without layout but maybe